src/dfmpreview/package.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "DfmPreview",
+    "displayName": "DfmPreview",
+    "description": "preview the DocFX Flavored Markdown in vscode",
+    "version": "0.0.7",
+    "publisher": "928pjy",
+    "engines": {
+        "vscode": "^1.3.0"
+    },
+    "categories": [
+        "Other"
+    ],
+    "activationEvents": [
+        "onCommand:DFM.showpreview",
+        "onCommand:DFM.showpreviewToside",
+        "oncommand:DFM.showsource"
+    ],
+    "main": "./out/src/extension",
+    "contributes": {
+        "commands": [
+            {
+                "command": "DFM.showpreview",
+                "title": "Toggle Preview"
```

Comment:
It conflicts with the default markdown commands. Besides, you may want to bind keys to these commands.